### DISCUSSION TITLE
is there any nmap output?

### DISCUSSION THREAD

**Discussion by t101804** [projectdiscovery/naabu#1196] [General]
### Please describe your feature request:
Hello currently im using naabu with --nmap-cli. and i want the nmap-cli get outputted bcs i using -oN in the nmap-cli and wont working

### Describe the use case of this feature:

naabu --host "$1" -v -p - -nmap-cli 'nmap -sV -Pn -oN $1-nmap-output.port' -o "$1-opened.port"

**Comment by GeorginaReeder** [comment]
Thanks for your feature request @t101804 , we'll take a look into this!

**Comment by Mzack9999** [comment]
@t101804 I'm unable to reproduce, I guess it might be connected to environment variable of your terminal:

```console
$ export aa=scanme.sh
$ naabu --host "$aa" -v -p - -nmap-cli 'nmap -sV -Pn -oN $aa-nmap-output.port' -o "$aa-opened.port"
...
$ ls 
scanme.sh-nmap-output.port  scanme.sh-opened.port
```
